cmd/utils/flags.go

```diff
@@ -71,6 +71,11 @@ var (
 	ConfFlag = cli.StringFlag{
 		Name: "conf",
 	}
+	UnsafeDebugDisableFlag = cli.BoolFlag{
+		Name:   "unsafe-debug.disable",
```

Comment:
There are other flags starting with "rpc". How about changing its name to "rpc.debug.disable-critical" or "rpc.debug.disable-unsafe"?

Reply:
Make sense. but I was thinking about the disable convention xxx.disable for hierarchy after applied yaml configuration. ref. [example.yaml](https://github.com/klaytn/klaytn/pull/1684/files#diff-6c1ff5ac872c0749f63ab487947ccc6140b5b9ee5c1b692de5c1511121b4d44cR15).

Then how about `rpc.unsafe-debug.disable`?

Reply:
Looks good to me